distribution/docs/src/main/resources/configuring-managed-service-factory-bundles-contents.adoc

```diff
@@ -1,99 +0,0 @@
-
```

Comment:
this file was a copy error. it should be removed from other branches as well. (The _real_ one is in `src/main/resources/_contents`)